### PR TITLE
MOSIP-28488

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: charts/apitestrig/templates/cronjob.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
   schedule: {{ $.Values.crontime }}
   jobTemplate:
     spec:
-      backoffLimit: 2 # this has very low chance of failing, as all this does
+      backoffLimit: 0 # this has very low chance of failing, as all this does
```
Comment (syedsalman3753) [inline charts/apitestrig/templates/cronjob.yaml:18]: Please templatize as you have done for other jobs

File: charts/dslorchestrator/templates/cronjob.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ spec:
   schedule: {{ .Values.crontime }}
   jobTemplate:
     spec:
-      backoffLimit: 2 # this has very low chance of failing, as all this does
+      backoffLimit: 0 # this has very low chance of failing, as all this does
```
Comment (syedsalman3753) [inline charts/dslorchestrator/templates/cronjob.yaml:16]: Please templatize as you have done for other jobs

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
Add PR message along with ticket ID

File: charts/dslorchestrator/templates/cronjob.yaml (hunk 2 above)
Comment (Mohanraj209) [reply charts/dslorchestrator/templates/cronjob.yaml:16]: done

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**